cachi2/core/package_managers/gomod.py

```diff
@@ -392,7 +395,7 @@ def _retry(self, cmd: list[str], **kwargs: Any) -> str:
         n_tries = get_config().gomod_download_max_tries
 
         @backoff.on_exception(
-            backoff.expo,
+            fast_constant_delay,
```

Comment:
What's the reasoning behind dropping exponential? Why is it deemed bad in our case? I'm genuinely curious because expo seemed to me to be the "industry" standard when it comes to dealing with networking responses.